### ISSUE TITLE
[Housekeeping] Traceback does not use the full width of the screen

### ISSUE THREAD

**Issue by thomasjpfan** [flyteorg/flyte#4488]
### Describe the issue

Tracebacks do not use the full width of the terminal screen. For example, with the following script:

```python
from flytekit import task


@task
def an_error():
    1 / 0

```

And running: `pyflyte run raise_error.py an_error` results this UI:

![Screenshot 2023-11-27 at 1 09 18 PM](https://github.com/flyteorg/flyte/assets/5402633/9f8a345f-6703-46ea-931f-98db91b7b026)

There is a lot of unused space for the trackback, which means we need to scroll up more to get to get to the location of the error.

### What if we do not do this?

One needs to scroll up more to get to the informative traceback.

### Related component(s)

flytekit

### Are you sure this issue hasn't been raised already?

- [X] Yes

### Have you read the Code of Conduct?

- [X] Yes

**Comment by thomasjpfan** [comment]
No, the bot's answer is not useful. We are using rich to control the width, which is under flytekit's control. The reason why the trackback is narrow is because there are several columns, where the first column is currently used for datetime info.

**Comment by pagetable** [comment]
#take
